schemas/x509.yml

```diff
@@ -0,0 +1,154 @@
+      level: extended
+      type: keyword
+      description: Country (C) code
+      example: US
+
+    - name: signature_algorithm
+      level: extended
+      type: keyword
+      description: Identifier for certificate signature algorithm. Recommend using names found in Go Lang Crypto library (https://golang.org/src/crypto/x509/x509.go).
+      example: SHA256-RSA
+
+    - name: valid_date
+      level: extended
+      type: date
+      description: Time at which the certificate is first considered valid.
+      example: 2019-08-16T01:40:25Z
+
+    - name: expire_date
+      level: extended
+      type: date
+      description: Time at which the certificate is no longer considered valid.
+      example: 2020-07-16T03:15:39Z
+
+    - name: subject.distinguished_name
+      level: extended
+      type: keyword
+      description: Distinguished name (DN) of the certificate subject entity.
+      example: C=US, ST=California, L=San Francisco, O=Fastly, Inc., CN=r2.shared.global.fastly.net
+
+    - name: subject.common_name
+      level: extended
+      type: keyword
+      description: Common name (CN) of subject.
+      example: r2.shared.global.fastly.net
+
+    - name: subject.organizational_unit
+      level: extended
+      type: keyword
+      description: List of organizational units (OU) of subject.
+
+    - name: subject.organization
+      level: extended
+      type: keyword
+      description: Organizational units (O) of subject.
+      example: Fastly, Inc.
+
+    - name: subject.locality
+      level: extended
+      type: keyword
+      description: Locality name (L)
+      example: San Francisco
+
+    - name: subject.state_or_province
+      level: extended
+      type: keyword
+      description: State or province name (ST, S, or P)
+      example: California
+
+    - name: subject.country
+      level: extended
+      type: keyword
+      description: Country (C) code
+      example: US
+
+    - name: public_key_algorithm
+      level: extended
+      type: keyword
+      description: Algorithm used to generate the public key.
+      example: RSA, ECDSA
+
+    - name: public_key_size
+      level: extended
+      type: long
+      description: The size of the public key space in bits.
+      example: 2048
+
+    - name: public_key_exponent
+      level: extended
+      type: long
+      description: Exponent used to derive the public key. This is algorithm specific.
+      example: 65537
+
+    - name: public_key_curve
+      level: extended
+      type: keyword
+      description: The curve used by the elliptic curve public key algorithm. This is algorithm specific.
+      example: nistp521
+
+    - name: alternative_names
+      level: extended
+      type: keyword
+      description: List of subject alternative names (SAN). Name types vary by certificate authority and certificate type but commonly contain IP addresses, DNS names (and wildcards), and email addresses.
+      example: "*.elastic.co"
```

Comment:
Please mark this field as normalize/array.

Please add a `short:` description here as well. Once again the first sentence looks perfect for the purpose.